pyproject.toml

```diff
@@ -49,6 +49,12 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
+force_single_line = true
+honor_noqa = true
+lines_between_types = 1
+profile = "black"
+reverse_relative = true
```

Comment:
This one needs motivation.

Reply:
After our discussion, I understood we will have a follow on PR immediately after this one to remove `reverse_relative` not that the circular deps checks are in place. @webknjaz I see this as resolved can you hit the button?

Reply:
related to this: https://github.com/ansible/ansible-navigator/issues/780

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback.  The follow-up PR was noted in the PR description and issue logged.